docker-compose.yml

```diff
@@ -124,16 +124,6 @@ services:
       - 9090:80
     command: sh -c "php -S 0.0.0.0:80 index.php"
 
-  agent:
```

Comment:
I use this `agent` target when I need to start up an agent to send traces to Datadog so I can look around in the UI. Is there any particular reason to drop this?